Framework/Algorithms/src/ReflectometryReductionOneAuto2.cpp

```diff
@@ -1016,8 +1016,9 @@ auto ReflectometryReductionOneAuto2::getOutputNamesForGroups(
   std::string informativeName = "TOF" + runNumber + "_";
 
   WorkspaceNames outputNames;
-  if (equal(informativeName.begin(), informativeName.end(),
-            inputName.begin())) {
+  if ((informativeName.length() < inputName.length()) &&
```

Comment:
This if statement is checking if informativeName is a substring of inputName eg
- informativeName=TOF_1234
- inputName = TOF_1234_Dannytest

...so I think this needs to stay as a < instead of = ? The call to std::equal checks that the characters in both strings up to the length of the informativeName are equal rather than full equality

Reply:
Ah okay, I misinterpreted the std::equal as check the full length of the name, rather than a substring